src/arch/riscv64/CodeGen.zig

```diff
@@ -49,7 +47,7 @@ const InnerError = CodeGenError || error{OutOfRegisters};
 
 gpa: Allocator,
 air: Air,
-mod: *Package.Module,
+mod: *Package.Zcu,
```

Comment:
nice find and replace :P
Please do at least look over diffs for automated changes like this before PRing, even if they seem trivial!

Reply:
Bruh, I could swear I looked over it. Thank you.